StatPlugin.cs

```diff
@@ -32,10 +33,16 @@ public HTTPResponse GetResponse(HTTPRequest request)
       HTTPResponse response = null;
       StringBuilder sb = new StringBuilder();
       sb.Append("<html><body><h1>Stat:</h1>");
-      foreach (KeyValuePair<String, int> entry in statDictionary)
-      {
-        sb.Append(entry.Key + ": " + entry.Value.ToString() + "<br />");
-      }
+      // foreach (KeyValuePair<String, int> entry in statDictionary)
```

Comment:
Don't remove the code by comment it out. It's ok to comment it out when you want to develop your code, but before you commit, you should delete all code commenting.